main.go

```diff
@@ -225,30 +226,26 @@ func convertField(curPkg *ProtoPackage, desc *descriptor.FieldDescriptorProto, m
 
 	opts := desc.GetOptions()
 	if opts != nil && proto.HasExtension(opts, protos.E_Bigquery) {
-		rawOpt, err := proto.GetExtension(opts, protos.E_Bigquery)
-		if err != nil {
-			return nil, err
-		}
-		opt := *rawOpt.(*protos.BigQueryFieldOptions)
-		if opt.Ignore {
+		opt := proto.GetExtension(opts, protos.E_Bigquery).(*protos.BigQueryFieldOptions)
+		if opt.GetIgnore() {
 			// skip the field below
 			return nil, nil
 		}
 
-		if opt.Require {
+		if opt.GetRequire() {
 			field.Mode = "REQUIRED"
 		}
 
-		if len(opt.TypeOverride) > 0 {
-			field.Type = opt.TypeOverride
+		if to := opt.GetTypeOverride(); len(to) > 0 {
+			field.Type = to
 		}
 
-		if len(opt.Name) > 0 {
-			field.Name = opt.Name
+		if n := opt.GetName(); len(n) > 0 {
+			field.Name = n
 		}
 
-		if len(opt.Description) > 0 {
-			field.Description = opt.Description
+		if d := opt.GetDescription(); len(d) > 0 {
```

Comment:
There is a well-known abbreviation for “description”.
```suggestion
		if desc := opt.GetDescription(); desc != "" {
```

Reply:
Done